implements.py

```diff
@@ -44,9 +44,13 @@ def implements(interface_cls):
     defined by the `interface_cls`.
     """
     def _decorator(cls):
-        verify_methods(interface_cls, cls)
-        verify_properties(interface_cls, cls)
-        verify_attributes(interface_cls, cls)
+        errors = []
+        errors.extend(verify_methods(interface_cls, cls))
+        errors.extend(verify_properties(interface_cls, cls))
+        errors.extend(verify_attributes(interface_cls, cls))
+        if len(errors):
+            raise NotImplementedError('Found errors in interfaces:\n'
```

Comment:
Recommendation:
Change 'Found errors in interfaces:' to 'Found {} errors in implementation:', with '{}' formatted to print len(errors)

Reply:
I think the change is a good idea. Does this look good?

```py
raise NotImplementedError(
    'Found {} errors in implementation:\n- {}\nwith {}'.format(
        len(errors), '\n- '.join(errors), cls))
```

Outputs:

```
NotImplementedError: Found 2 errors in implementation:
- 'MallardDuck' must implement method 'fly(self) -> bool' defined in interface 'Quackable'
- 'MallardDuck' must implement method 'quack(self)' defined in interface 'Quackable'
with <class '__main__.MallardDuck'>
```